pnpm-lock.yaml

```diff
@@ -6764,7 +6800,7 @@ packages:
     engines: {node: ^14.15.0 || ^16.10.0 || >=18.0.0}
     dependencies:
       '@jest/types': 29.6.3
-      '@types/node': 20.10.8
+      '@types/node': 18.19.14
```

Comment:
y tho

Reply:
Not sure, but I ran another `pnpm dedupe` and it went away.